rbc/tests/test_omnisci_math.py

```diff
@@ -0,0 +1,418 @@
+                'select x, {fn_name}(x) from {omnisci.table_name}'
+                .format(**locals()))
+
+            for x, v in list(result):
+                assert(np.isclose(np_fn(x), v))
+
+
+def test_hyperbolic_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('double(double)')  # noqa: F811
+    def _sinh(x):
+        return np.sinh(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def _cosh(x):
+        return np.cosh(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def _tanh(x):
+        return np.tanh(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def arcsinh(x):
+        return np.arcsinh(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def arccosh(x):
+        return np.arccosh(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def arctanh(x):
+        return np.arctanh(x)
+
+    omnisci.register()
+
+    for fn_name in ['_sinh', '_cosh', '_tanh', 'arcsinh',
+                    'arccosh', 'arctanh']:
+        np_fn = getattr(np, fn_name.lstrip('_'))
+
+        query = ''
+        if fn_name == 'arccosh':
+            query = 'select i, {fn_name}(i) from {omnisci.table_name}'\
+                    .format(**locals())
+        else:
+            query = 'select x, {fn_name}(x) from {omnisci.table_name}'\
+                    .format(**locals())
+
+        descr, result = omnisci.sql_execute(query)
+
+        for x, v in list(result):
+            assert(np.isclose(np_fn(x), v))
+
+
+def test_rounding_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('double(double)')  # noqa: F811
+    def around(x):
+        return np.around(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def round_(x):
+        return np.round_(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def _rint(x):
+        return np.rint(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def floor(x):
+        return np.floor(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def ceil(x):
+        return np.ceil(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def _trunc(x):
+        return np.trunc(x)
+
+    omnisci.register()
+
+    for fn_name in ['around', '_rint', 'round_', 'floor', 'ceil', '_trunc']:
+        np_fn = getattr(np, fn_name.lstrip('_'))
+
+        query = ''
+        if fn_name == 'arccosh':
+            query = 'select i, {fn_name}(i) from {omnisci.table_name}'\
+                    .format(**locals())
+        else:
+            query = 'select x, {fn_name}(x) from {omnisci.table_name}'\
+                    .format(**locals())
+
+        descr, result = omnisci.sql_execute(query)
+
+        for x, v in list(result):
+            assert(np.isclose(np_fn(x), v))
+
+
+def test_explog_funcs(omnisci):
+    omnisci.reset()
+
+    @omnisci('double(double)')  # noqa: F811
+    def exp(x):
+        return np.exp(x)
+
+    @omnisci('double(double)')  # noqa: F811
+    def _expm1(x):
+        return np.expm1(x)
+
+    # doesn't work - even adding a prefix '_'
```

Comment:
Does it work when using other prefixes? Say, `_np`?
Open a separate rbc issue for this.